src/frontends/tensorflow_lite/tests/requirements.txt

```diff
@@ -0,0 +1,2 @@
+numpy>=1.16.6,<=1.23.4
+tensorflow>=1.15.5,<=2.11.0
```

Comment:
let's also add this file to dependably config https://github.com/openvinotoolkit/openvino/blob/master/.github/dependabot.yml

Reply:
Done